src/menu/menu.h

```diff
@@ -7,9 +7,15 @@
 #ifndef MENU_H__
 #define MENU_H__
 
+#include <libdragon.h>
 
 #include "boot/boot.h"
 
+extern wav64_t sfx_cursor, sfx_error, sfx_enter, sfx_exit, sfx_settings;
+
+enum sfx_channels {
+	SFX_CHANNEL
```

Comment:
This uses the exactly same mixer channel as MP3 player. Also this should be moved to the `sound.h` file.